chingu-voyages/v52-tier3-team-32
# changes to Admin interface

# Changes made:
1. Re-arranged the codebase folder structure
2. Added metadata relevant images, i.e., opengraph-image.jpg, icon.jpg, twitter-image.jpg, apple-icon.jpg
3. Refactored code in admin interface by renaming file paths to list view be distict to maps view to reports view
4. Implemented an aesthetic navigation bar (by shadcn-ui), in the admins interface.
5. Added a map provided by [`React-Leaflets`](https://react-leaflet.js.org/) library.

# How to test: `via the cli`
1. Clone the repository,
2. Run the command `cd web-app` for unix shell users
3. Check into the branch `feature/admin_interface`; i.e., `git checkout -b feature/admin-interface`.
4. Run `pnpm install` or `npm install`
5. Run `prisma generate`.
6. Run `pnpm dev` or `npm run dev`
7. view the UI in your browser, and interact with it.

## Notes:
Enjoy testing...
